### DISCUSSION TITLE
TrinityCore3.3.5 npc names mistaken code problem

### DISCUSSION THREAD

**Discussion by Binarycocaine** [TrinityCore/TrinityCore#28769] [Q&A]
Hi , I'm use TrinityCore 3.3.5 and launch the server well

But when login the game,I found npc's names are just unreadable,and type simple-Chinese characters just ok

![image](https://user-images.githubusercontent.com/26806225/213620686-cce0dd60-4d50-40cd-9509-9ae98e9b8518.png)

![image](https://user-images.githubusercontent.com/26806225/213621165-d6bf1ef6-63dc-4c9f-94d8-cf66e9b4f2e3.png)

![image](https://user-images.githubusercontent.com/26806225/213621651-3b5ba3a3-16ee-46bd-a1c5-efbb11eb2898.png)

I'm using a Simple-Chinese client.

just wondering how to fix this problem, thank you!

**Comment by CraftedRO** [comment]
Learn english and then change your client to EN/GB or EN/US.

**Reply by Faq** [reply]
Or wait till someone with same client provides a fix.

**Comment by Aokromes** [comment]
i vote he imported the database instead leave core to import it.

**Reply by Jinnaix** [reply]
Pretty sure it's this, happens with HeidiSQL for example, nukes all local table content.

**Reply by CraftedRO** [reply]
I'm also always import "by hand" all three databases (auth, characters and world) first time when I'm installing on clean, and never had a problem and yea using heidi.

**Reply by Jinnaix** [reply]
Call yourself lucky in that case. Manually importing the database is simply not suggested.

**Reply by Shauren** [reply]
HeidiSQL is known to cause exactly this issue (destroys locale tables, english stays fine)

**Reply by CraftedRO** [reply]
Iteresting is that database inserted by core take up less space than the one inserted by heidi:
Inserted by core:
![database inserted with core](https://user-images.githubusercontent.com/24683355/213788786-674cb318-8415-436b-9167-9d6613a98175.PNG)

Iserted by heidi:
![database inserted with heidi](https://user-images.githubusercontent.com/24683355/213788858-2ed2816c-a1c6-4b61-b16a-1b7945f6fb64.PNG)
